### ISSUE TITLE
Top banner display for Org branding 

### ISSUE THREAD

**Issue by stephaniesimms** [DMPRoadmap/roadmap#526]
Related issue #525 
UX wireframe @rhu001 - 

Display Org details in top banner if provided (optional)
- Replace DMPonline/DMPTool logo w/Org logo 
- DMPonline/DMPTool logo still displayed in footer
- Include Org full name, actionable URL, and actionable admin contact email as in DMPTool 
- Add field to include icon / link for RDM policy

Roadmap Org details page and banner display
![org-details](https://user-images.githubusercontent.com/12703638/28435087-8ac23392-6d46-11e7-8aae-bde77d206197.png)

DMPTool Institutional Profile page and banner display
![screen shot 2017-07-20 at 12 03 17 pm](https://user-images.githubusercontent.com/12703638/28435106-968a26d0-6d46-11e7-9e71-bf3f0b1bff50.png)

**Comment by sjDCC** [comment]
Just copying ideas over that we added to discussion thread in #424. We hadn't seen you'd done a wireframe already - sorry. This link to it is broken at moment so if you could redirect we'll review before call.

We discussed how to handle institutional logos today and have a suggestion to make. We'd prefer not to lose the DMPonline one completely, but don't feel two logos work well side-by-side so in effect agree with the decision to replace this with the institutional one. What we'd propose as an addition is switching the DMPonline logo to be an icon at the start of the menu. We started to mock-up a visual of this. 

One change layout wise would be dropping the menu beneath the primary logo rather than it running adjacent to it (bottom aligned) as this would allow for different sizes (square, landscape orientation etc) of uni logos more easily.

![uni-logo-example](https://user-images.githubusercontent.com/6169940/28791652-0afa3ff0-7625-11e7-95fd-9be7a5a199bd.png)

@stephaniesimms I'm not entirely convinced by giving the institutional name in full. What's the purpose of this? It may look odd when we remove the DMPTool / DMPonline logo as their institutional presence will be main thing on tool

**Comment by stephaniesimms** [comment]
many logos for US unis don't include the institution name so we need to include it somewhere. and we're fine w/institutional presence being more prominent since for us the goal of institutional branding is to encourage users to contact their local admins rather than our helpdesk for DMP support. our branding is much less important since it doesn't matter to us if users understand that CDL maintains the DMPTool.

**Comment by sjDCC** [comment]
Ok, that makes sense then. Most of our uni logos do have the name included rather than just a crest.

@briri Could you confirm that field is optional and if people leave it blank then a name doesn't display in middle? If so, that's perfect as it meets both use cases.

We definitely want people to contact local admins rather than us too, but still want some brand recognition for DMPonline so people know that's what the tool is. @rhu001 @JEK-III - can you consider how this could be worked into the design. Perhaps the icon idea we suggested above, or moving logo to footer or other suggestion?

**Comment by stephaniesimms** [comment]
we definitely want dmptool represented in the footer (added to requirements above). this issue was just for institutional branding, which is why it wasn't noted here. the position of the institution name is negotiable - we just need the option to display it somewhere.

**Comment by briri** [comment]
That shouldn't be an issue @sjDCC. Right now we're displaying the "Top Banner Text" in that area. Should we consider switching that to a regular text field rather than one of the tinymce boxes? This might deter org admins from entering really long paragraphs of text if what we ideally want is just their name or a tag line?

**Comment by sjDCC** [comment]
Thanks both.

@briri - the suggestion was to remove the top banner text. I'm happy for this to be replaced with a plain text field for org name. The only thing some of our unis record there which isn't in your current model is a link to the RDM policy so I suggest we introduce an icon for this. In ticket #525 @vyruss listed all the existing data for top banner text. I added this to your overaching list at the top @stephaniesimms 

If you're adding the DMPTool logo to the footer then I think that's a compromise we can reach. We don't need the DMPonline logo in the top banner, but we don't want it to disappear completely, hence suggesting adding it as an icon to the front of the menu. The way we've replaced it on other instances would suit. Screengrab below for reference @rhu001 @JEK-III 

![melbourne](https://user-images.githubusercontent.com/6169940/28881782-2b7e7fca-77a1-11e7-9737-c825422ebdee.PNG)
